### PR TITLE
add cctv traj

### PR DESCRIPTION
**Pull request by odufour7**
- I modified the name of the cameras in the .json file to match the MADRAS article
- I added a new camera to the map the one at the exit of place des terreaux to match the MADRAS article
- I added a tab about visualisation in time and in space of trajectories on the place des terreaux. The data of cameras from airbnb just need to be updated to the right coordinate system (this can be added later in the python file also)
- I modified the contact_tab to make it more friendly and readable. 
- I still have to included analysis of CCTV cameras

### CODE REVIEW

File: src/tabs/animation_tab.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import streamlit as st
 import plotly.express as px
 from plotly.graph_objs import Figure
+from pyproj import Transformer
```
Comment (chraibi) [inline src/tabs/animation_tab.py:7]: did you add pyproj to requirements.txt?

File: src/tabs/animation_tab.py [hunk 2]
```diff
@@ -18,10 +19,44 @@ def load_data(pickle_name: str) -> pd.DataFrame:
         pd.DataFrame: DataFrame containing the pedestrian trajectory data.
     """
     pd_trajs = pd.read_pickle(pickle_name)
+    print(pd_trajs.head())
```
Comment (chraibi) [inline src/tabs/animation_tab.py]: remove print statement

File: src/tabs/contacts_tab.py [hunk 3]
```diff
@@ -102,25 +102,14 @@ def add_contact_markers(map_object: folium.Map, contact_gps_merged: pd.DataFrame
...
-    Args:
-        df (pd.DataFrame): The DataFrame containing the 'Total-number-of-collisions' column.
-
-    Returns:
-        Figure: The Plotly figure object for the histogram.
-    """
-  
-    fig, ax = plt.subplots(figsize=(2, 2), dpi=10)
-    sns.histplot(df['Total-number-of-collisions'], bins=bins, kde=True, log_scale=(log_plot[0], log_plot[1]), ax=ax)
+def plot_histogram(df: pd.DataFrame, bins: int, log_plot: Tuple[bool, bool]) -> plt.Figure:
```
Comment (chraibi) [inline src/tabs/contacts_tab.py:394]: docstring missing

**Review by chraibi — CHANGES_REQUESTED** [review]

File: src/tabs/animation_tab.py (hunk 1 above)
Comment (odufour7) [reply src/tabs/animation_tab.py:7]: not yet

**Comment by odufour7** [comment]
Ok, I have made the changes

**Pull request closed without merging**